input/fsh/SD_HistoryOfMetastaticCancer.fsh

```diff
@@ -10,4 +10,4 @@ Id: mcode-history-of-metastatic-malignant-neoplasm-vs
 Title: "History of Metastatic Malignant Neoplasm Value Set"
 Description: "Values defining history of metastatic cancer."
 * insert SNOMEDCopyrightForVS
-* include codes from system SCT where concept is-a #1287652008 "History of metastatic malignant neoplasm (situation)"
+* include codes from system SCT_TBD where concept is-a #1287652008 "History of metastatic malignant neoplasm (situation)"
```

Comment:
I made this change in the other pull request.
I left the intentional definition as a comment, since when the code is issued we want the intensionality in SCT.